.github/workflows/python-test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

Comment:
The Python versions should be declared as strings as in the [docs of the setup python action](https://github.com/actions/setup-python#usage), otherwise `3.10` might be interpreted as `3.1` in the workflow.